aggregator/aggregator.go

```diff
@@ -51,7 +51,7 @@ const (
 func RunWithConfig(configPath string) error {
 	nodeConfig, err := config.NewConfig(configPath)
 	if err != nil {
-		panic(fmt.Errorf("failed to parse config file: %w\nMake sure %s is exist and a valid yaml file %w.", configPath, err))
+		panic(fmt.Errorf("failed to parse config file: %s\nMake sure it is exist and a valid yaml file %w.", configPath, err))
```

Comment:
Let’s fix the capitalization now - the first letter of a sentence must be capitalized. Please fix other instances of fmt.Errorf() too. 

Use `Failed to`, not `failed to`.

Do the below return return the message string to the gRPC client?
```
return fmt.Errorf("failed to parse config file: %s\nMake sure it is exist and a valid yaml file %w.", configPath, err)
```